.eslintignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 build
 reports
+lib
```

Comment:
add a page break for each of these little red nope guys ⛔

Reply:
fixed: https://github.com/cerner/terra-toolkit/pull/430/commits/c5f53239e40409db99c5768be53dfc102d5a4531